src/utils/messages.ts

```diff
@@ -0,0 +1,7 @@
+export const USER_EMAIL_NOT_FOUND = 'User with Email {0} not found';
```

Comment:
- khai báo rồi export cả thảy luôn em ơi
- file message thì em dùng chữ thường khai báo message là đc nha, ko cần dùng in hoa để phân biệt với constant ở file constants 
vd:
```
const user_mail_not_found = 'User with Email {0} not found';
const user_mail_existed = 'Email already existed';
...
export {user_mail_not_found, user_mail_existed}
```

Reply:
Dạ vâng chị